.jscsrc

```diff
@@ -1,4 +1,10 @@
 {
   "preset": "airbnb",
-  "excludeFiles": ['dist']
+  "excludeFiles": ['dist'],
+  "requireSpacesInsideObjectBrackets": false,
+  "requireCamelCaseOrUpperCaseIdentifiers": {
+    allExcept: [
+      "child_process",
```

Comment:
Super nitpick, but we should use trailing commas consistently if we're to use them.

(Personally I prefer not to as these configs are basically JSON and JSON would say "no trailing commas".)

But we should at least be consistent; this has a trailing comma, but the enclosing array and object don't.